index.html

```diff
@@ -453,12 +453,11 @@ <h2>The <code><dfn>IdleDeadline</dfn></code> interface</h2>
       milliseconds of the deadline. This MUST be populated when the
       <a>IdleDeadline</a> is created.</p>
       <p>When the <code><dfn>timeRemaining</dfn>()</code> method is invoked on
-      an <a>IdleDeadline</a>
-      object it MUST return the duration, as a <a>DOMHighResTimeStamp</a>,
-      between the current time and the <a>time</a> associated with the
-      <a>IdleDeadline</a> object. The value SHOULD be accurate to 5 microseconds
-      - see "Privacy and Security" section of [[!HR-TIME]]. This value is
-      calculated by performing the following steps:</p>
+      an <a>IdleDeadline</a> object it MUST return the remain duration before
+      the deadline expires as a <a>DOMHighResTimeStamp</a>, which SHOULD be
+      accurate to 5 microseconds - see "Privacy and Security" section of
```

Comment:
I believe the 5 microsecond language is not longer true in a post-Spectre world.

Reply:
Updated to "enough to allow measurement while preventing timing attack" based on hr-time text.

Reply:
:+1: